src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -64,6 +74,10 @@ public class HttpProtocolTests : HttpClientTest
         [InlineData("HTTP/1.1 2345")]
         [InlineData("HTTP/A.1 200 OK")]
         [InlineData("HTTP/X.Y.Z 200 OK")]
+        // The following disabled by ActiveIssue: 26542
+        //[InlineData("HTTP/1.1\t200 OK")]
+        //[InlineData("HTTP/1.1 200\tOK")]
+        //[InlineData("HTTP/1.1 200\t")]
```

Comment:
And same for this bug - let's mark it with disabled-test label once your change gets in.

Reply:
Added disabled-test label.

Reply:
>Added disabled-test label.

I think @karelz was referring to adding that label to the original GitHub issue *after* this PR gets merged but not before.

Reply:
Correct, but the timing doesn't make much difference ;)